src/interface/report/CombatantInfoFaker.ts

```diff
@@ -137,6 +137,8 @@ const MAIL_SPECS = [
   SPECS.ELEMENTAL_SHAMAN.id,
   SPECS.ENHANCEMENT_SHAMAN.id,
   SPECS.RESTORATION_SHAMAN.id,
+  SPECS.DEVASTATION_EVOKER.id,
```

Comment:
I'm not 100% but I'm pretty certain that this whole class can and probably should be deleted and it was hyper specific to shadowlands beta
@Pewtro Should know since he was the creator of it

Reply:
- Adds new class (game class) 
- Gets confused about this comment

Took me a while to understand haha :)

Reply:
If combatantinfo isn’t available once scripts are enabled then it could be useful again, but for now probably no need to add it unless it starts breaking stuff

Reply:
Do I need to make changes to the CombatantnfoFaker?

Reply:
No, I’ll do a pass at it later in the cycle once scripts are enabled to see if it has merit or not.

Reply:
we currently have combatantinfo, but no talent info in there yet.